feeds/wlan-ap/opensync/files/bin/wlan_ap_rrm_rebalance.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+filename="/tmp/wireless_changes.txt"
+freq_table="1:2412 2:2417 3:2422 4:2427 5:2432 6:2437 7:2442 8:2447 9:2452 10:2457 11:2462 12:2467 13:2472 14:2484 36:5180 40:5200 44:5220 48:5240 52:5260 56:5280 60:5300 64:5320 100:5500 104:5520 108:5540 112:5560 116:5580 120:5600 124:5620 128:5640 132:5660 136:5680 140:5700 144:5720 149:5745 153:5765 157:5785 161:5805 165:5825"
+while IFS= read -r line; do
+        if [ "$line" = "phy0" ]; then
```

Comment:
There must be a more elegant way to do this.  Hard-coding the number of radios is not great.